api/v1alpha1/openapi.yaml

```diff
@@ -4025,7 +4025,15 @@ components:
         agentVersion:
           type: string
           description: The Agent version.
+        facts:
```

Comment:
I know naming is hard, but "facts" seems to generic to me, and may also imply that other information returned is not facts?

Reply:
I took the naming from Ansible[1], open to other ideas `details` maybe? But I believe we want the name fairly generic to also accommodate custom user defined fields.

[1] https://www.redhat.com/en/blog/playing-ansible-facts

Reply:
I see, got it.  Yea, "details" or "attributes" seem less loaded.

Reply:
cc @fzdarsky

Reply:
Between these 3 choices, I like "attributes".

Reply:
i thought that would not be legal but I will explore